src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/ConfirmationStatementServiceEligibilityConfig.java

```diff
@@ -18,14 +19,19 @@
     @Value("${ALLOWED_COMPANY_STATUSES}")
     Set<String> allowedCompanyStatuses;
 
+    @Value("${ALLOWED_COMPANY_TYPES}")
+    Set<String> allowedCompanyTypes;
```

Comment:
As the validation for this story is ruling out a specific subset of companies that don't need to file a CS01.
May be best for this to be a Disallow set.
Something like companyTypesNotRequiredToFileCS01 (or something more eloquent)
And if the type of the company in question is inside that set. Fail validation